frontend/app/routes/_gcweb-app.letters._index.tsx

```diff
@@ -61,27 +61,23 @@ export default function LettersIndex() {
 
   return (
     <div>
-      <label htmlFor="sort-order" className="text-sm">
+      <label htmlFor="sort-order" className="block text-sm font-medium text-gray-900">
```

Comment:
Keep font color same as the document.

```suggestion
      <label htmlFor="sort-order" className="block text-base font-medium">
```